dcos/config.py

```diff
@@ -213,6 +213,26 @@ def get_config_schema(command):
     return subcommand.config_schema(executable, command)
 
 
+def get_property_description(name, value):
+    """
+    :param name: name of paramater
+    :type name: str
```

Comment:
`name` is the config section or subcommand. We call it section in the rest of the code. `param` is the property within the section. We call it subkey in the rest of the code. Take a look at line 27 in this file.